src/main/kotlin/no/nav/dagpenger/data/innlop/tjenester/SøknadIdentRiver.kt

```diff
@@ -29,6 +31,9 @@ internal class SøknadIdentRiver(
     override fun onPacket(packet: JsonMessage, context: MessageContext) {
         val søknadId = packet["søknad_uuid"].asUUID()
         val ident = packet["ident"].asText()
+        val person = personRepository.hentPerson(ident)
+
+        if (person.harAdressebeskyttelse) return
```

Comment:
Funksjonen slutter (returnerer ingenting) slik at dataene ikke blir sendt til kafka-topic når personen har adressebeskyttelse?

Reply:
Yep.

Denne kortslutter bare inserten vi gjør i ident-tabellen da. Så man vil ikke kunne mappe en søknad til person med adressebeskyttelse, men søknadsdataen vil jo fortsatt komme inn.

Så man kan jo utlede at om man ikke får mappet så er det en søknad fra en person med adressebeskyttelse, men skal jo fortsatt godt gjøres å koble det til *EN* person.

Reply:
Skjønner og enig i at det vil være vanskelig basert på dataene fra søknad som er lagt i dataproduktet. Er jo andre felter som kan brukes til å identifisere hvis en har tilgang til andre kilder (journalpostId og fagsakId). Men det vil bare være mulig å utlede at søknaden er kode 6 og koble med de andre feltene hvis en har tilgang til tabellene som laget viewet og ikke viewet. Så risiko er redusert. 

Tipper det er greit for @caroliwi siden de ikke lenger får personer med kode 6.